src/current/_includes/common/network-partitions.md

```diff
@@ -0,0 +1,7 @@
+A network partition occurs when two or more nodes are prevented from communicating with each other in one or both directions. A network partition can be caused by a network outage or a configuration problem with the network, such as when allowlisted IP addresses or hostnames change after a node is torn down and rebuilt.
+
+In a **symmetric** partition, node communication is disrupted in both directions. In an **asymmetric** partition, nodes can communicate in one direction but not the other.
+
+CockroachDB protects against asymmetric partitions by converting all asymmetric (uni-directional) network partitions into symmetric (bi-directional) network partitions. This increases cluster resiliency by reducing the number of partition-related failures that can occur.  Many temporary symmetric partitions can be recovered from automatically without operator intervention.
+
+The effect of a network partition depends on which nodes are partitioned, where the ranges are located, and to a large extent, whether [localities]({% link {{ page.version.version }}/cockroach-start.md %}#locality) are defined. If localities are not defined, a partition that cuts off at least `(n-1)/2` nodes will cause data unavailability.
```

Comment:
This last statement regarding data unavailability is true when applied to a single range, but in general there if are many ranges then the bound is lower in practice. 

I think it might be more informative to say something like:

> The effect of a network partition depends on which nodes are partitioned, where the ranges are located, and to a large extent, whether [localities]({% link {{ page.version.version }}/cockroach-start.md %}#locality) or zone configurations are defined. A partition that cuts off at least `(n-1)/2` replicas from a range will cause range unavailability which will cause some data unavailability. If there are no localities or other constraints on where replicas are places, then a partition of any RF/2 nodes will likely cause unavailability.

Reply:
thanks for the updated text @andrewbaptist

I have updated the final paragraph with your suggested language, edited slightly to add links to stuff (like changing the mention of 'RF' to link to [the docs for `num_replicas` in zone configs](https://www.cockroachlabs.com/docs/stable/configure-replication-zones.html#num_replicas) for example)

PTAL!